pkg/subnet/upgrades/upgrade_file.go

```diff
@@ -37,25 +51,31 @@ func WriteUpgradeFile(jsonBytes []byte, subnetName string, app *application.Aval
 		}
 	}
 
-	if err = os.WriteFile(upgradeBytesFilePath, jsonBytes, constants.DefaultPerms755); err != nil {
-		return err
-	}
-	ux.Logger.PrintToUser("File written successfully")
-	return nil
+	return os.WriteFile(path, jsonBytes, constants.DefaultPerms755)
 }
 
 func ReadUpgradeFile(subnetName string, app *application.Avalanche) ([]byte, error) {
 	localUpgradeBytesFilePath := app.GetUpgradeBytesFilePath(subnetName)
 
-	exists, err := storage.FileExists(localUpgradeBytesFilePath)
+	return readFile(localUpgradeBytesFilePath)
+}
+
+func ReadLockUpgradeFile(subnetName string, app *application.Avalanche) ([]byte, error) {
```

Comment:
Same comment as above. Use the app package to read the file, do type conversions and return a mature data type, not []byte